src/components/core/ErrorModal.js

```diff
@@ -26,7 +27,72 @@ const formatError = (error) => {
     newError = newError.replace(text, '');
   });
 
-  return newError;
+  let translatedError = '';
+  switch(newError) {
```

Comment:
Looks like something's adding a leading space to at least some of the error codes making this lead to the default case with everything I've tested with.
newError.trim() fixes it, but maybe it would be worth it tracing it to the source.